gcp_568_translator.py

```diff
@@ -0,0 +1,78 @@
+import json
```

Comment:
We'll want a couple unit tests for this just showing that a successful translation is published, and a failed translation results in error messages.